cli/command/new/cmd.go

```diff
@@ -76,9 +76,10 @@ func Command(cli *command.SauceCtlCli) *cobra.Command {
 func Run(cmd *cobra.Command, cli *command.SauceCtlCli, args []string) error {
 	creds := credentials.Get()
 	if creds == nil {
-		fmt.Println("\nIt looks you have not configured your SauceLab account !" +
-			"\nTo enjoy SauceLabs capabilities, configure your account by running:\n$ saucectl configure")
-		return nil
+		log.Error().Msg("\nsaucectl requires a valid Sauce Labs account to run."+
```

Comment:
Might be better to not use the logger at all in this case, but straight up `fmt.Println()`. If we want colored output, consider using https://github.com/fatih/color here 💡